cli-commands/shape/shape.js

```diff
@@ -0,0 +1,41 @@
+const logger = require('../../logger-service/logger-service').logger;
+const git = require('simple-git/promise')()
+const util = require('util');
+const exec = util.promisify(require('child_process').exec);
+const reposUrl = require('./urlConfig').reposUrls;
```

Comment:
I do not like the name of this variable. I'd name it something like `repoToUrlMap`